internal/logstreamer/logstreamer.go

```diff
@@ -0,0 +1,146 @@
+package logstreamer
```

Comment:
Did you consider using a git submodule?

Reply:
No, not really. I'd rather keep the git repo simple, we have no other submodules either. It's a simple package and this way we can easily modify to our needs.